snap/snapcraft.yaml

```diff
@@ -26,34 +33,34 @@ apps:
       - x11
```

Comment:
You might want to add `opengl` to plugs. I get errors when it's not added, it seems krop tries to use the gpu.